### PR TITLE
Fix incorrect return value

### PR DESCRIPTION
**Pull request by buthed010203**
the javadoc is a lie

- [x] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [ ] I have ensured that my code compiles, if applicable.
- [ ] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/src/mindustry/net/Administration.java [hunk 1]
```diff
@@ -302,11 +302,13 @@ public boolean unbanPlayerID(String id){
     public boolean adminPlayer(String id, String usid){
         PlayerInfo info = getCreateInfo(id);
 
+        if(info.admin) return true;
```
Comment (Anuken) [inline core/src/mindustry/net/Administration.java:305]: Might not want to return early here, so the usid is updated.

Comment (buthed010203) [reply core/src/mindustry/net/Administration.java:305]: ah true

**Comment by buthed010203** [comment]
fixed

**Comment by buthed010203** [comment]
Idk why i wrapped it in newlines, i really shouldnt have but whatever

**Pull request merged**